values.yaml

```diff
@@ -219,6 +219,28 @@ global:
     # Requires consul-k8s 0.15.0+.
     createFederationSecret: false
 
+  # Resource settings for lifecycle-sidecar containers
+  lifecycleSidecarContainerResources:
```

Comment:
I'm wondering if we should add another level here:
```
lifecycleSidecar:
  resources:
```

I think that will future proof us if we want to add more settings to the lifecycle sidecar in the future (like disabling it).